spec/board_spec.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require './lib/board'
+require './lib/validation'
+
+describe Board do
+  let(:tictactoe) { Board.new }
+
+  describe '#update_board' do
+    it 'updates the position 1 with an X' do
+      tictactoe.update_board('X', 1)
+      expect(tictactoe.board[0]).to eql('X')
+    end
+  end
```

Comment:
- Here You can test the case when the user chooses an already occupied cell or an invalid cell position.

Reply:
Thank you for your feedback, but if you check the validation_spec.rb you will see that we have already handled that case.